code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -52,6 +52,11 @@ def __init__(self, **kwargs) -> None:
         self.AZURE_SEARCH_CONVERSATIONS_LOG_INDEX = os.getenv(
             "AZURE_SEARCH_CONVERSATIONS_LOG_INDEX", "conversations"
         )
+        # Integrated Vectorization
+        self.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION = (
```

Comment:
There's actually a function for this below, `get_env_var_bool`

Reply:
Updated